map_model/src/objects/building.rs

```diff
@@ -26,6 +26,7 @@ impl fmt::Display for BuildingID {
 pub struct Building {
     pub id: BuildingID,
     pub polygon: Polygon,
+    pub levels: f64,
```

Comment:
I was curious if we could get away with `usize`, but https://taginfo.openstreetmap.org/keys/building%3Alevels#values shows fractional values.

Net file size change from this PR is 3MB, nice!